<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="ScannetrabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>560</width>
     <height>581</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>560</width>
     <height>540</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="ScannerTab">
    <attribute name="title">
     <string>Scanner</string>
    </attribute>
    <widget class="QTreeWidget" name="ScannerTreeWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>541</width>
       <height>501</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>SSID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>BSSID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ENCRTPT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CHANNEL</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SIGNAL</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STATIONs</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DATAs</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="ChannelLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>channel</string>
     </property>
    </widget>
    <widget class="QLabel" name="StatuLabel">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>status</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="AnalyzerTab">
    <attribute name="title">
     <string>@-@</string>
    </attribute>
    <widget class="QTreeWidget" name="AnalyzerTreeWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>541</width>
       <height>501</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="StatuLabel_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>statu1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="StartButton">
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StopButton">
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>ScannetrabWidget</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>StatuLabel</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
